content/en/blog/_posts/2025-01-22-kube-scheduler-simulator/index.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: blog
```

Comment:
Do you want to refer the article: https://www.cncf.io/blog/2022/08/24/writing-your-own-scheduler-with-kube-scheduler-simulator/? Not sure what is the relashinship here between articles

Reply:
I didn't know that article, and I don't want to, honestly.